infra/services/image-store/ingress.yaml

```
@@ -8,7 +8,7 @@ spec:
   rules:
     - http:
         paths:
-          - path: /images
+          - path: /image-api/images
```

Should only match `image-api` part

```suggestion
          - path: /image-api
```